JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -238,7 +238,7 @@ private fun InterestScreenContent(
 private val tabContainerModifier = Modifier
     .fillMaxWidth()
     .wrapContentWidth(Alignment.CenterHorizontally)
-    .navigationBarsPadding(start = false, end = false)
+    .navigationBarsPadding()
```

Comment:
This isn't a direct replacement, but due to consumption, this actually will act the same: the start and end won't be applied again, since they are already being applied in `JetnewsApp`